ui/src/app/applications/components/application-details/application-resource-list.tsx

```diff
@@ -102,9 +102,15 @@ export const ApplicationResourceList = (props: ApplicationResourceListProps) =>
                                             </Consumer>
                                         )}
                                     </div>
-                                    <div className='columns small-1 xxxlarge-1'>{[res.group, res.kind].filter(item => !!item).join('/')}</div>
-                                    <div className='columns small-1 xxxlarge-1'>{res.syncWave || '-'}</div>
-                                    <div className='columns small-2 xxxlarge-1'>{res.namespace}</div>
+                                    <div className='columns small-1 xxxlarge-1' title={[res.group, res.kind].filter(item => !!item).join('/')}>
```

Comment:
let's move this filtering to local variable, to reuse code

Reply:
Thanks for your review! I applied your review and defined a local variable to reuse.
df59d795364acff62460f67a65c17b62df7d2fe4